Documentation/Installation.rst

```diff
@@ -9,8 +9,8 @@ Building the package
 --------------------
 
 
-* Ensure that 'dpkg' and 'svn' packages are installed on the machine
-* Check out the root of the branch/tag/branch of MCVirt
+* Ensure the build dependancies are installed: ``dpkg, python-docutils``

```

Comment:
Probably not worth mentioning for now. It won't really be handled correctly if docutils is missing, since an ImportError will be printed to stderr in the build output.